pkg/plugin/linuxutil/netstat_stats_linux.go

```diff
@@ -205,21 +192,10 @@ func (nr *NetstatReader) readSockStats() error {
 		// Log the socketByRemoteAddr map
 		if nr.opts.PrevTCPSockStats != nil {
```

Comment:
I'd suggest deleting this `PrevTCPSockStats` from the repo if there's no chance of using it in the future

Reply:
or we could file an issue to clean it up later

Reply:
I have removed it.